packages/jaeger-ui/src/model/ddg/types.tsx

```diff
@@ -41,12 +43,15 @@ export type TDdgPath = {
 
 export type TDdgServiceMap = Map<string, TDdgService>;
 
-export type TDdgPathElemsByDistance = Map<number, PathElem[]>;
+export type TDdgDistanceToPathElems = Map<number, PathElem[]>;
+
 export type TDdgVisibilityIdxToPathElem = Map<number, PathElem>;
 
 export type TDdgModel = {
-  pathElemsByDistance: TDdgPathElemsByDistance;
+  distanceToPathElems: TDdgDistanceToPathElems;
   paths: TDdgPath[];
   services: TDdgServiceMap;
   visibilityIdxToPathElem: TDdgVisibilityIdxToPathElem;
 };
+
+export type TDdgEdgeIdentifier = 'egressEdges' | 'ingressEdges';
```

Comment:
Why not use an enum, here?

Reply:
tbh I forget `enum` exists! we never used before because this "works"
but since this case is exactly what enums are for, updated.